packages/documentation/src/stories/components/textarea/textarea.stories.ts

```diff
@@ -165,7 +169,9 @@ function renderTextarea(args: Args, context: StoryContext) {
       ?disabled=${args.disabled}
       aria-label=${useAriaLabel ? args.label : nothing}
       aria-invalid=${VALIDATION_STATE_MAP[args.validation] ?? nothing}
-      aria-describedby="${args.hint ? 'form-hint-' + context.id : ''}"
+      aria-describedby="${args.hint ? 'form-hint-' + context.id : ''} ${args.validation != 'null'
+        ? `${args.validation}-id-${context.id}`
+        : ''}"
```

Comment:
Can you compute this in an array, then use `.join(' ')` to avoid having unwanted spaces in the snippet?
![image](https://github.com/user-attachments/assets/1ccc80bb-0229-44c0-a553-c8681846a532)